docs/root/intro/arch_overview/redis.rst

```diff
@@ -57,6 +57,30 @@ If passive healthchecking is desired, also configure
 For the purposes of passive healthchecking, connect timeouts, command timeouts, and connection
 close map to 5xx. All other responses from Redis are counted as a success.
 
+Redis Cluster Support (Experimental)
+----------------------------------------
+
+Envoy currently offers experimental support for `Redis Cluster <https://redis.io/topics/cluster-spec>`_.
+
+Why do we need a new system? Beside the expected benefits of a sidecar
```

Comment:
This doesn't read quite right now that you got rid of the twemproxy stuff. Can you wordsmith/rephrase slightly so it just talks about Redis Cluster directly and not a new system, etc.